mk/clang.mk

```diff
@@ -17,11 +17,13 @@ CFLAGS_DEBUG=-g
 
 ifeq ($(OSTYPE),darwin)
 ARCH=$(shell uname -m)
+PARTIALLD=ld64 -r -all_load
```

Comment:
wat, ld64 doesnt exist in mac

Reply:
Oops, I forgot that it's only the codebase that's called ld64, not the binary